addon/components/loading-scroll-view.js

```diff
@@ -0,0 +1,67 @@
+import Ember from 'ember';
+import ScrollView from './scroll-view';
+const { observer } = Ember;
+
+const THRESHOLD_FROM_BOTTOM_TO_TRIGGER_LOAD_MORE = 350;
+const MAX_LOAD_MORE_FREQUENCY_MS = 1000;
+
+export default ScrollView.extend({
+  scrollableRegistry: null, //injected
```

Comment:
Since it's already injected with the mixin, I reckon we can just remove the line outright, no?